app/api/event_copyright.py

```diff
@@ -40,22 +40,33 @@ class EventCopyrightList(ResourceList):
 
     def query(self, view_kwargs):
```

Comment:
@SaptakS Implementing for other APIs too. Check WIP [here](https://github.com/enigmaeth/open-event-orga-server/commits/exceptn). Will make a PR in a few hours. Just added changes for these two here.